src/diffusers/schedulers/scheduling_edm_dpmsolver_multistep.py

```diff
@@ -295,6 +294,8 @@ def _compute_exponential_sigmas(self, ramp, sigma_min=None, sigma_max=None) -> t
         sigma_min = sigma_min or self.config.sigma_min
         sigma_max = sigma_max or self.config.sigma_max
         sigmas = torch.linspace(math.log(sigma_min), math.log(sigma_max), len(ramp)).exp().flip(0)
+        if not torch.is_tensor(ramp):
```

Comment:
we make the `ramp` variable here and it is a numpy 
```python
ramp = np.linspace(0, 1, self.num_inference_steps)
```

Reply:
Sorry, I didn't get your point. 

We compute `ramp` using `torch.linspace()` here:
https://github.com/huggingface/diffusers/blob/4d633bfe9a3830dfd50b3f70d8469a575dbaf75a/src/diffusers/schedulers/scheduling_edm_euler.py#L103

And then we compute it with `np.linspace()` here:
https://github.com/huggingface/diffusers/blob/4d633bfe9a3830dfd50b3f70d8469a575dbaf75a/src/diffusers/schedulers/scheduling_edm_euler.py#L213

Reply:
we make `ramp` we can decide what they are - so there is no need to have the `if` here :)